test/jdk/java/util/Properties/PropertiesStoreTest.java

```diff
@@ -46,12 +47,18 @@
 /*
  * @test
  * @summary tests the order in which the Properties.store() method writes out the properties
- * @bug 8231640
- * @run testng PropertiesStoreTest
+ * @bug 8231640 8282023
+ * @modules jdk.localedata
+ * @run testng/othervm PropertiesStoreTest
  */
 public class PropertiesStoreTest {
 
     private static final String DATE_FORMAT_PATTERN = "EEE MMM dd HH:mm:ss zzz uuuu";
+    // use a neutral locale, since when the date comment was written by Properties.store(...),
+    // it internally calls the Date.toString() which always writes in a locale insensitive manner
+    private static final DateTimeFormatter formatter = DateTimeFormatter.ofPattern(DATE_FORMAT_PATTERN)
+            .withLocale(Locale.ROOT);
```

Comment:
Should have noticed before, but you can call the convenient overload of `ofPattern(DATE_FORMAT_PATTERN, Locale.ROOT)` here.

Reply:
That makes sense. I hadn't noticed that API before. I've updated the PR with this suggested change.